com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/ValidateDeployParametersStep.java

```diff
@@ -48,32 +58,72 @@ protected String getStepErrorMessage(DelegateExecution context) {
     }
 
     private void validateParameters(DelegateExecution context) {
-        validateStartTimeout(context);
-        validateAppArchiveId(context);
-        validateExtDescriptorFileId(context);
-        validateVersionRule(context);
+        try {
+            validateStartTimeout(context);
+            validateAppArchiveId(context);
+            validateExtDescriptorFileId(context);
+            validateVersionRule(context);
+            verifyArchive(context);
+        } finally {
+            deleteArchiveFile(context);
+        }
     }
 
-    private void validateVersionRule(DelegateExecution context) {
-        String versionRuleString = (String) context.getVariable(Constants.PARAM_VERSION_RULE);
+    private void validateStartTimeout(DelegateExecution context) {
+        Object parameter = context.getVariable(Constants.PARAM_START_TIMEOUT);
+        if (parameter == null) {
+            return;
+        }
+        int startTimeout = (Integer) parameter;
```

Comment:
Why not cast directly to the primitive type?

Reply:
Done.